gzip_stream.py

```diff
@@ -8,14 +8,16 @@
 
 
 class GZIPCompressedStream(io.RawIOBase):
-    def __init__(self, stream: BinaryIO, *, compression_level: int):
+    def __init__(self, stream: BinaryIO, *, compression_level: int,
+                 file_name: str = None):
```

Comment:
I think, it'll better to name this argument `filename` to avoid confusion (same as it named in the `gzip.GzipFile` class).
```suggestion
                 filename: str = None):
```